.github/workflows/cd.yml

```diff
@@ -0,0 +1,30 @@
+name: CD
+
+on: [release]
+
+jobs:
+  publish-chart:
+    runs-on: ubuntu-latest
+    needs: test-chart
+    if: "startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master'"
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: set up environment variables
+      run: |
+        echo "GIT_USER=Renku Bot" >> $GITHUB_ENV
+        echo "GIT_EMAIL=renku@datascience.ch" >> $GITHUB_ENV
+    - name: Push chart and images
+      uses: SwissDataScienceCenter/renku/actions/publish-chart@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.DEPLOY_TOKEN }}
```

Comment:
should we make this an org-level secret?

Reply:
yep, seems like a good idea.